packages/safe-ds-vscode/src/extension/mainClient.ts

```diff
@@ -99,9 +103,9 @@ const registerMessageLoggingCallbacks = function () {
         printOutputMessage(
             `Placeholder was calculated (${message.id}): ${message.data.name} of type ${message.data.type}`,
         );
-        const execInfo = getExecutionContext(message.id)!;
-        execInfo.calculatedPlaceholders.set(message.data.name, message.data.type);
-        sendMessageToPythonServer(createPlaceholderQueryMessage(message.id, message.data.name));
+        // const execInfo = getExecutionContext(message.id)!;
+        // execInfo.calculatedPlaceholders.set(message.data.name, message.data.type);
+        // sendMessageToPythonServer(createPlaceholderQueryMessage(message.id, message.data.name));
```

Comment:
The request of placeholder values is not needed anymore, but I thought storing the calculated placeholder names with their types in the pipeline context could be helpful for future work or debugging purposes

Reply:
I see yes, that may make sense, I will uncomment the lines 106 and 107